entries/varooom3d/game.json

```diff
@@ -12,7 +12,6 @@
             "playable": true
         }
     ],
-    "license": "GPL-3.0-or-later",
```

Comment:
@GValiente why are we removing this? Under which terms is the game released?

Reply:
It's complicated :)

Game's source code is part of Butano, licensed under Zlib.
Third party libraries are licensed under other similar licenses like MIT.
Assets like graphics and music are licensed under multiple CC licenses, some of them non commercial.

Nothing is licensed under GPL, though.

Reply:
Okay, that makes sense. We do support different licences for assets and code but we don't have a supported field yet for "miscellanea"..
I can suggest you to put the more limitative license (among the multiple ones that apply to the assets) in the `assetsLicense` field, while you can put zlib under the `gameLicense` field